jest.config.js

```diff
@@ -1,20 +1,5 @@
+// This is a root level jest config for running tests
+// across all packages.
 module.exports = {
-  preset: 'ts-jest',
-  setupFilesAfterEnv: ['./jest.setup.ts'],
-  clearMocks: true,
-  restoreMocks: true,
-  testMatch: [
-    '<rootDir>/src/**/*.test.ts',
-    '<rootDir>/test/**/*.test.ts',
-    '<rootDir>/maintenance-jobs/**/*.test.ts',
-    '!**/*.bak/*',
-  ],
-  testPathIgnorePatterns: ['<rootDir>/dist/', '<rootDir>/node_modules/'],
-  collectCoverage: false,
-  globals: {
-    'ts-jest': {
-      isolatedModules: true,
-    },
-  },
-  testEnvironment: 'node',
+  projects: ['<rootDir>/packages/*'],
```

Comment:
This allows for tests to be run from the root while allowing each package to still have their own test configuration.